example/pubspec.lock

```diff
@@ -21,7 +21,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "1.1.0"
+    version: "1.1.1"
```

Comment:
Just confirming: this a a version behind because the example lock file doesn't know about the new 1.2 yet, right?

Reply:
correct, that always lags one version behind until we publish the version update change during the release script